SamKTech/Sanum
# user's goals are being set, middleware updated

Co-author: @khuongdn16 

## **Types of Changes**


## **Purpose**
- being able to set calorie goals for a user

## **Approach**
- utilizing Context API to pass state from parent component App.jsx to Profile.jsx file  

## **How to Test**
- npm run dev + after logging in input calories target/goal on your profile page and save with confirm button (the calorie goal amount should render on the page right below the input window)

## **Screenshot(s)**
(bold descriptions)

<img width="1680" alt="Screenshot 2021-04-01 at 04 53 25" src="https://user-images.githubusercontent.com/64405107/113237081-79c34980-92a6-11eb-9e34-2921614e0b2a.png">
<img width="1680" alt="Screenshot 2021-04-01 at 04 53 41" src="https://user-images.githubusercontent.com/64405107/113237089-7d56d080-92a6-11eb-81ec-0c0055210e8e.png">
